ui/apps/platform/scripts/start-ocp-console.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
+CONSOLE_PORT=${CONSOLE_PORT:=9000}
+CONSOLE_IMAGE_PLATFORM=${CONSOLE_IMAGE_PLATFORM:="linux/amd64"}
```

Comment:
Are `CONSOLE_WHATEVER` conventions for scripts in similar plugins?

My first impression is that `CONSOLE` has many meanings.

Concerning `BRIDGE` below, I am trying to remember if there was a similar concept for cloud service UI?

Reply:
Yes, `CONSOLE_` is what is used in the upstream template for this file: https://github.com/openshift/console-plugin-template/blob/main/start-console.sh although I don't think that we need to be consistent with those files. It might be helpful to keep as-is in case we ever need to refer back to the original.

For `BRIDGE_`, I am not sure if there is a cloud service equivalent. Those variables must be named as-is though, as they are used by the console container that is executed at the end of the script.